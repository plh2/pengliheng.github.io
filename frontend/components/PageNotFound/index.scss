.PageNotFound {
	width: 100%;
	display: flex;
	align-items: center;
	justify-content: center;

	img {
		width: 80%;
	}
}
