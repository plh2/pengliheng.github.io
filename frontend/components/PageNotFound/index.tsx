import React from "react";
import NotFoundSvg from "@/assets/404_img.svg";
import Image from "next/image";
import "./index.scss";

const PageNotFound = () => (
  <div className="PageNotFound">
    <Image alt="not_found" src={NotFoundSvg} />
  </div>
);

export default PageNotFound;
