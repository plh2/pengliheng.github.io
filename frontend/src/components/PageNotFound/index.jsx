import React from 'react'
import './index.less'
import NotFoundSvg from '@/ASSETS/404_img.svg'

export default () => (
	<div className="PageNotFound">
		<img alt="not_found" src={NotFoundSvg} />
	</div>
)
