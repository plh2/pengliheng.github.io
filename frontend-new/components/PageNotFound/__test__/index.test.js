// Link.react.test.js
import React from 'react'
import PageNotFound from '../index.jsx'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
	const tree = renderer.create(<PageNotFound />).toJSON()
	expect(tree).toMatchSnapshot()
})
