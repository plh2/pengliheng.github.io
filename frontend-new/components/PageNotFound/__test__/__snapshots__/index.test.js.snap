// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="PageNotFound"
>
  <img
    alt="not_found"
    src={Object {}}
  />
</div>
`;
